src/Core/RulesEventManager.php

```diff
@@ -60,4 +60,22 @@ public function processDefinition(&$definition, $plugin_id) {
     }
   }
 
+  /**
+   * Returns the base name of a configured event name.
+   *
+   * For a configured event name like node_view--article the base event name
```

Comment:
oudated examples, check configurable event interface and also add an @see to it as this holds the related docs